src/SFA.DAS.Commitments.Application/Queries/GetApprenticeships/GetApprenticeshipsQueryHandler.cs

```diff
@@ -28,14 +28,14 @@ public GetApprenticeshipsQueryHandler(IApprenticeshipRepository apprenticeshipRe
             };
         }
 
-        private async Task<ApprenticeshipsResult> GetApprenticeships(Caller caller, string searchKeyword)
+        private async Task<ApprenticeshipsResult> GetApprenticeships(Caller caller)
         {
             switch (caller.CallerType)
             {
                 case CallerType.Employer:
-                    return await _apprenticeshipRepository.GetApprenticeshipsByEmployer(caller.Id, searchKeyword);
+                    return await _apprenticeshipRepository.GetActiveApprenticeshipsByEmployer(caller.Id);
                 case CallerType.Provider:
-                    return await _apprenticeshipRepository.GetApprenticeshipsByProvider(caller.Id, searchKeyword);
```

Comment:
ApprenticeshipRepository.GetApprenticeshipsByProvider can be removed as it is no longer called.